pennylane/qnodes/base.py

```diff
@@ -167,25 +165,23 @@ def __init__(self, func, device, *, mutable=True, properties=None):
         self.func = func  #: callable: quantum function
         self.device = device  #: Device: device that executes the circuit
         self.num_wires = device.num_wires  #: int: number of subsystems (wires) in the circuit
-        self.num_variables = (
-            None  #: int: number of flattened differentiable parameters in the circuit
-        )
+        #: int: number of flattened differentiable parameters in the circuit
```

Comment:
Does Sphinx properly extract `#:` style docstrings if they are above the attribute? An alternate is to use a multiline string below the attribute.

Reply:
That's annoying :/